apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx

```diff
@@ -14,7 +14,7 @@ export const FluentDocsContainer: React.FC<FluentDocsContainerProps> = ({ childr
   return (
     <>
       {/** TODO add table of contents */}
-      <FluentProvider style={{ backgroundColor: 'transparent' }} theme={webLightTheme}>
+      <FluentProvider className="sb-unstyled" style={{ backgroundColor: 'transparent' }} theme={webLightTheme}>
```

Comment:
Without this particular class, Storybook applies numerous default styles to common HTML elements such as headings, lists, paragraphs, and more. Since it utilizes CSS-in-JS, overriding these styles with traditional CSS proves challenging.

<img width="1444" alt="Screenshot 2024-08-04 at 11 18 38" src="https://github.com/user-attachments/assets/7ec0b559-8a4c-4d72-9453-afb68ca452e5">

Reply:
gotcha, lets add this context to the source as comment pls. ty